doc/release-notes-26328.md

```diff
@@ -0,0 +1,7 @@
+Tools and Utilities
+-------------------
+
+- CLI `-netinfo` now displays 3 different states in the `txn` column (true, false, and not available), as the
+  latter value is a new, third state since 24.0 returned by the RPC getpeerinfo "relaytxes" field when the peer
```

Comment:
(cont'd)

```suggestion
  last value is a new third state since 24.0 returned by the RPC getpeerinfo "relaytxes" field when the peer
```

Reply:
Thanks, replaced "latter" with "not available".